src/RenderingComponent.h

```diff
@@ -7,18 +7,22 @@
 #include "SDL_opengl.h"
 #include "glm/glm.hpp"
 
-#include "MeshAsset.h"
+class MeshAsset;
+class TextureAsset;
+class ShaderAsset;
 
 class RenderingComponent {
 public:
-    //Takes a list of camera matrices and renders up to 4 views
-    //Does not set the shader
-    void render_views(std::array<glm::mat4x4, 4>& cameras, size_t count, GLuint program_id);
+    RenderingComponent();
+    //Renders the object from the camera perspective.
+    void render(glm::mat4x4 camera) const;
 
     void apply_transform(glm::mat4x4 transform);
     void set_transform(glm::mat4x4 transform);
```

Comment:
I know it's not part of this PR, but can we comment on the difference between these two functions?